events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,333 @@
+
+#######################
+# Trapped Residence
+#######################
+stasis_trap_outcome.2001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.2001.desc
+
+		first_valid = {
+			# If the Scheme Owner is known:
+			triggered_desc = {
+				trigger = { 
+					OR = {
+						exists = scope:scheme_discovered
+					}
+				}
+				desc = stasis_trap_outcome.2001.owner_is_known
+			}
+			desc = stasis_trap_outcome.2001.no_awareness
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = fear
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_success_effect = yes
+	}
+
+	option = {
+		name = stasis_trap_outcome.2001.a
+		play_music_cue = "mx_cue_prison"
+		successful_stasis_trap_outcome_event_option_effect = yes
+	}
+}
+
+#################################
+# FAILURE OUTCOMES 4000 - 4999
+#################################
+stasis_trap_outcome.4001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.4001.opening
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.4001.owner_discovered
+			}
+			desc = stasis_trap_outcome.4001.owner_secret
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = rage
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_failure_effect = yes
+	}
+
+	option = { # Abandon scheme
+		name = stasis_trap_outcome.4001.a
+		abandon_scheme_effect = yes
+
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = { # Try again!
+		name = stasis_trap_outcome.4001.b
+		restart_scheme_effect = yes
+
+		ai_chance = {
+			base = 10
+			modifier = {
+				add = 200
+				has_trait = stubborn
+			}
+		}
+	}
+}
+
+########################################
+# TARGET SUCCESS NOTIFICATIONS
+########################################
+
+#I have been stunned!
+stasis_trap_outcome.5001 = {
+	type = character_event
+	title = stasis_trap_outcome.5001.t
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.5001.opening.b
+			}
+			triggered_desc = {
+				trigger = {
+					NOT = {
+						exists = scope:scheme_discovered
+					}
+				}
+				desc = stasis_trap_outcome.5001.opening.a
+			}
+		}
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:stasis_trap_method = flag:trapped_residence
+				}
+				desc = stasis_trap_outcome.5001.trapped_residence
+			}
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:owner
+		animation = scheme
+	}
```

Comment:
You show the scheme owner here though they may be unknown. The issue comes from the abduction events where the owner is always known if the scheme is succesful.

Reply:
Solved in 
5975e49